### DISCUSSION TITLE
Batch import .mesh files

### DISCUSSION THREAD

**Discussion by Kaspr02** [ross-g/io_pdx_mesh#37] [Ideas]
Would it be too much of a hassle to add a multiple files import? 
Also, an option to name the objects after the .mesh filename (except .mesh extension). This would make importing all the blendshapes easier.

**Comment by ross-g** [comment]
I like the suggestion and you're certainly not the only one wanting to import multiple assets into one scene. (see #28) And blendshapes are indeed a pain to manage at the moment.

It's not really possible to name the imported objects after the filename, as a file can contain many objects. In fact the mesh files have one object for each material used on a source mesh (ie meshes are split by material on export). So if you made a cube with 2 materials on, export it and import into a new scene... you will get two half cubes back.

Is it just blendshapes that you'd want to use this for? I have been toying with the idea of an import option to pull in blendshapes and set them up as shape-keys / morph-targets automatically.
![image](https://user-images.githubusercontent.com/9258768/107132103-1df6c880-68d4-11eb-8a70-ed866c5c8757.png)

**Reply by Kaspr02** [reply]
Yes it's just the blendshapes I'd want to use this for.  An import option for this would be a relief. There are hundreds of blendshape files, this would take forever if done manually.
